packages/clay-table/src/Body.tsx

```diff
@@ -0,0 +1,26 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import * as React from 'react';
+
+interface BodyProps
+	extends React.TableHTMLAttributes<HTMLTableSectionElement> {}
+
+export type TableBodyType = React.FunctionComponent<BodyProps>;
+
+const Body: React.FunctionComponent<BodyProps> = ({
+	children,
+	className,
+	...otherProps
+}) => {
+	return (
+		<tbody {...otherProps} className={className}>
```

Comment:
No need to declare className here. It'll already be there with `...otherProps`

Reply:
Fixed